cmake/modules/AddCephTest.cmake

```diff
@@ -2,7 +2,35 @@
 
 #adds makes target/script into a test, test to check target, sets necessary environment variables
 function(add_ceph_test test_name test_path)
-  add_test(NAME ${test_name} COMMAND ${test_path} ${ARGN}
+  # crimson seastar unittest always run with --smp N
+  string(FIND "${ARGN}" "--smp" smp_pos)
```

Comment:
seastar::smp::count is determined based on `--cpuset` size provided. We can avoid setting both of them.

Reply:
> seastar::smp::count is determined based on --cpuset size provided 

Yes,  but when we use `add_ceph_unittest(unittest-seastar-config --memory 256M --smp 4)` to add crimson test, we just know how many cores needed, it is complicated to decide `--cpuset`, so we add `--cpuset` automatically according `--smp`,  it is unnecessary to remove `--smp`  from the exists args

Reply:
seastar's `cpuset` defaults to "all" cores. Can we set only `--smp` in our tests?

Reply:
No, without `cpuset`, it will start from cpu0, if there are many jobs, cpu0 will be shared and running pretty slow.